python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -28,19 +28,22 @@
 
 from cugraph.utilities.utils import import_optional, MissingModule
 
-import dask.dataframe as dd
-from dask.distributed import get_client
-
+dd = import_optional("dask.dataframe")
+distributed = import_optional("dask.distributed")
+dask_cudf = import_optional("dask_cudf")
```

Comment:
Are these not always imported by cugraph, CC: @rlratzel  ?

Reply:
I believe these are always imported, but in anticipation of making those optional in cugraph in the future, I used `import_optional` here

Reply:
We have indeed been talking about the possibility of making the MG dependencies optional in cugraph.